typescript/karma.conf.js

```diff
@@ -29,8 +29,10 @@ module.exports = function (config) {
       subdir: '.',
       reporters: [
         { type: 'html' },
-        { type: 'text-summary' }
-      ]
+        { type: 'cobertura' },
+        { type: 'text-summary' },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/newsbridge/sandbox/pullRequest?prid=7541583)
```suggestion
        { type: "text-summary" },
```